src/Command/Initializer/Profiles/blank/Init.php

```diff
@@ -0,0 +1,612 @@
+                        "if (false) { // Should be set to true in production\n" .
+                        "    \$containerBuilder->enableCompilation(__DIR__ . '/../var/cache');\n}";
+                    $setContainerReplace = "// Build PHP-DI Container instance\n" .
+                        "\$container = \$containerBuilder->build();\n\n// Instantiate the app\n" .
+                        "AppFactory::setContainer(\$container);";
+                    break;
+                case PimpleDependency::class:
+                    $containerVariableReplace = '$container';
+                    $importsReplace = "use Pimple\Container;\nuse Slim\Factory\AppFactory;";
+                    $defineContainerReplace = "// Instantiate Pimple Container\n\$container = new Container();";
+                    $setContainerReplace = "// Instantiate the app\n" .
+                        "AppFactory::setContainer(new \Pimple\Psr11\Container(\$container));";
+                    break;
+                case OtherDependency::class:
+                    $containerVariableReplace = '$container';
+                    $importsReplace = "use Slim\Factory\AppFactory;";
+                    $defineContainerReplace = "// TODO: Instantiate you'r Dependency Container\n\$container = null;";
+                    $setContainerReplace = "// Instantiate the app\n" .
+                        "// TODO: Uncomment the line below if you created an instance of Dependency Container\n" .
+                        "//AppFactory::setContainer(\$container);";
+                    break;
+            }
+        }
+
+        return (new FileBuilder($templatePath))
+            ->setReplaceToken('{containerVariable}', $containerVariableReplace)
+            ->setReplaceToken('{imports}', $importsReplace)
+            ->setReplaceToken('{defineContainer}', $defineContainerReplace)
+            ->setReplaceToken('{setContainer}', $setContainerReplace)
+            ->buildFile($destinationFile);
+    }
+
+    /**
+     * Collect dependencies from user input.
+     *
+     * @return array<Dependency|null>
+     */
+    protected function askDependencies(): array
+    {
+        $psr7 = null;
+        $dependencyContainer = null;
+        $dependencyContainerOtherPackage = null;
+        $dependencyContainerOtherVersion = null;
+        $logger = null;
+        $availableDependencies = [
+            'psr7' => [
+                SlimPsr7Dependency::NAME => new SlimPsr7Dependency(),
+                LaminasDependency::NAME  => new LaminasDependency(),
+                GuzzleDependency::NAME   => new GuzzleDependency(),
+                NyholmDependency::NAME   => new NyholmDependency(),
+            ],
+            'dependencyContainer' => [
+                PHPDIDependency::NAME  => new PHPDIDependency(),
+                PimpleDependency::NAME => new PimpleDependency(),
+                OtherDependency::NAME  => new OtherDependency(),
```

Comment:
@l0gicgate now it only adds the composer's dependency. We ask for the package name and version:
![Screenshot_20200627_225240](https://user-images.githubusercontent.com/19558543/85929838-04ff5100-b8c9-11ea-82ae-e43f181a8f92.png)

Reply:
@ABGEO my question is more so after we install that unknown container dependency, we can’t add any of the boilerplate code that we are adding with PHP-DI and pimple. So this will just install that dependency and nothing else? Instead we should just maybe provide a 3rd option and remove that “Other” option?

Reply:
It also creates some TODOs in `index.php` because we don’t know what to add:
![Screenshot_20200627_230347](https://user-images.githubusercontent.com/19558543/85930108-aaff8b00-b8ca-11ea-9068-2d7ea166e828.png)

Reply:
I’m not a fan of this since it leaves the initial app in a broken state and people suck at reading. Let’s add “league/container” and “aura/di” and remove the “other” option.

Reply:
@ABGEO I think this might be the only last couple changes left before we can start writing tests. @adriansuter do you have any other changes you'd like to see?

Reply:
@l0gicgate I can't get the `league/container` to work with Slim. Do you have examples of `dependencies.php` and `settings.php`?

Reply:
Same thing with `aura/di`

Reply:
@ABGEO it will be different syntax for all packages.. you can't use `addDefinitions()` method. The only method that's standardized is the `get()` and `has()` method via PSR-11.

It's going to have to be a different `settings.php` and `dependencies.php` for each implementation